templates/02-executable-org.md

```diff
@@ -85,5 +87,6 @@ The number (#) should represent the number of proposals the Org has submitted to
 
 >❗️If applicable, please include a report and brief retrospective on spending throughout the previous year.
 
-## Fund Management
+## Management of Org Funds
 >👉 Outline fund management structure and protocol. How will Radworks funds be secured? How will contributors be paid out? At what cadence? How will Radworks funds be recovered in case of emergency?
```

Comment:
```suggestion
>👉 Outline fund management structure and protocol. How will Radworks' funds be secured? How will contributors be paid out? At what cadence? How will Radworks' funds be recovered in case of emergency?
```